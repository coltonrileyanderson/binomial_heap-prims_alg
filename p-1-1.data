3356 8629 307 ; 221 4507 21 ; 3272 9597 98 ; 6539 8643 14 ; 28 7897 434 ; 
2517 8514 67 ; 9477 5183 382 ; 3641 2592 243 ; 250 2716 0 ; 9371 7483 119 ; 

