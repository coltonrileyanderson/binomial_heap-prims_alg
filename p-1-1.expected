0: 3356
1: 8629(3356)307
weight: 307
